lib/data/external/AwsClient.js

```diff
@@ -39,16 +39,16 @@ class AwsClient {
         const putCb = (err, data) => {
```

Comment:
One more change of this nature is needed to remove "AWS" in the logs here: https://github.com/scality/S3/pull/1130/commits/80ab22592c97f3e5f9cd35e98ebf39ac884d2463#diff-bdedeaa43c674dd6fc4de3fdd0d030bcR109.